README.md

```
@@ -409,13 +409,19 @@ to see some common workflows on the quickstart dataset.
 
 Note that if you are running the above code in a script, you must include
 `session.wait()` to block execution until you close the App. See
-[this page](https://voxel51.com/docs/fiftyone/user_guide/app.html#creating-a-session)
+[this page](https://docs.voxel51.com/user_guide/app.html#creating-a-session)
 for more information.
 
 <div id='key-features'>
 
 ## <img src="https://user-images.githubusercontent.com/25985824/106288517-2422e000-6216-11eb-871d-26ad2e7b1e59.png" height="20px"> &nbsp; key features &nbsp; 🔑
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)